website/src/css/custom.css

```diff
@@ -4,27 +4,70 @@
  * work well for content-centric websites.
  */
 
+/* imported fonts */
+/* TODO: discuss if we want to use these or not */
+@font-face {
+  font-family: ShareTechMono;
+  src: url("../../static/fonts/Share_Tech_Mono.woff2");
+  font-weight: normal;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: DMMono;
+  src: url("../../static/fonts/DM_Mono.woff2");
+  font-weight: normal;
+  font-style: normal;
+}
+
 /* You can override the default Infima variables here. */
 :root {
-  --ifm-color-primary: #2e8555;
-  --ifm-color-primary-dark: #29784c;
-  --ifm-color-primary-darker: #277148;
-  --ifm-color-primary-darkest: #205d3b;
-  --ifm-color-primary-light: #33925d;
-  --ifm-color-primary-lighter: #359962;
-  --ifm-color-primary-lightest: #3cad6e;
+  --ifm-color-primary: #5e5e5e;
+  --ifm-background-color: #ffffff;
+  --ifm-navbar-background-color: #ffffff;
+  --ifm-footer-background-color: #020203;
+  --ifm-footer-color: #ffffff;
+  --ifm-footer-link-color: #ffffff;
+  --ifm-footer-link-hover-color: #ffffff;
+  --ifm-link-color: #579bea;
+  --ifm-color-hero: #fcfcfc;
+  /* TODO: discuss if we want to use this or not */
+  /* --ifm-font-family-base: DMMono; */
```

Comment:
Commented out font-family changes for now, pending discussion:
https://discord.com/channels/943612659163602974/1183821110844858462/1197273940075221163